configure.ac

```diff
@@ -1106,6 +1106,23 @@ OPAL_MCA
 
 m4_ifdef([project_ompi], [OMPI_REQUIRE_ENDPOINT_TAG_FINI])
 
+# Last minute disable of Open SHMEM if we didn't find any oshmem SPMLs
+if test "$project_oshmem_amc" = "true" -a $OSHMEM_FOUND_WORKING_SPML -eq 0 ; then
+    # We don't have an spml that will work, so oshmem wouldn't be able
+    # to run an application.  Therefore, don't build the oshmem layer.
+    if test "$enable_oshmem" != "no" -a -n "$enable_oshmem"; then
+	AC_MSG_WARN([No spml found, so Open SHMEM layer will be non functional.])
```

Comment:
Minor nit: unlike Open MPI, the name is actually "OpenSHMEM" (i.e., no space).

Reply:
I saw Open SHMEM more than OpenSHMEM in our build code; will fix.